logo, opengraph, and favicon updates

edgeworthe opened the pull request:
Various logo updates, and also reverts 00270a3 since it seemed to break Twitter DMs for me

Commit:
Ananse Ntontan favicon file

Based on JS's 8-bit adinkra SVG, converted to PNG and background
added via https://realfavicongenerator.net/

In case they retain the generated images there, should be
downloadable via:

https://realfavicongenerator.net/favicon_result?file_id=p1do9b1nbnuvl1vir1mu31p4p6sk6#.Xbb2yZNKh3k

Using android-chrome-192x192.png since it had the best fidelity.

Commit:
revert url encoding

I wasn't able to get url encoding to work with non-unicode
text, so reverting for now to unblock deploys.

Revert "refactors reason encoding for twitter DM, updates README for open source documentation"

This reverts commit 00270a3ab9f641746384181a2d661faaeede3806.

Commit:
new site logo

Has an issue where it doesn't display well on a white background,
and the margins are a little funky due to sizing mismatches, but
am hoping to get those fixed in next iteration.

Commit:
update favicon to 8-bit ananse ntontan

See http://www.adinkra.org/htmls/adinkra_index.htm
for inspiration.  Thanks to JS for the logo!

Commit:
update opengraph image to match our new site logo

Props to JS for the update and png-ification.

New logo includes more of the 8-bit adinkra images!

Commit:
update site logo to svg using dynamic textcolor

It took me a while to figure out the textcolor thing that
womenwho.design uses, but once we had an SVG with actual
text (thanks JS!), it was relatively straightforward to
restore the dynamic textcolor switching.

TheBrotherFromASouthernMother commented on src/pages/nominate.js:
```diff
@@ -54,12 +52,11 @@ class App extends React.Component {
     const validation = this.validateName();
-    if (!validation && window.confirm('This form will take you to Twitter https://twitter.com/ in order to submit your nomination')) {
+    if (!validation && window.confirm('This form will take you to Twitter https://twitter.com/ in order to send your nomination')) {
       const urlEncodedTwitterHandle = encode(this.state.name);
```
Looks like we can delete this variable `urlEncodedTwitterHandle ` and `urlEncodedReason` since we're replacing them with the properties in `this.state`

edgeworthe replied:
True, good point.  The changes here are only from reverting the one commit, so I didn't dig into further changes since I wasn't sure whether we'd retry the encoding changes or just roll back completely.  Given the short timeline currently, I'll look into clearing them out

TheBrotherFromASouthernMother replied:
Ahhh gotcha, that makes total sense!

TheBrotherFromASouthernMother commented:
These changes seem to have fixed the logo image margins!
![Screen Shot 2019-11-04 at 6 40 10 AM](https://user-images.githubusercontent.com/27319519/68129211-05785100-fece-11e9-9085-90aeff6f96c5.png)

edgeworthe commented on src/components/nav/nav.module.scss:
```diff
@@ -22,7 +22,7 @@
 .logo {
   display: block;
-  width: 150px;
+  width: 250px;
```
> These changes seem to have fixed the logo image margins!

It's this diff specifically that adjusted the margins by changing the width on the logo—I think there's still a _slight_ difference between the left and right margins on the logo, but @jayuhmee seemed ok with the current state, and I trust her take more than my own 😅

TheBrotherFromASouthernMother replied:
Ahhh honestly didn't even realize that until you pointed it out!

Commit:
remove unused variables and newer submit text

Variable changes suggested in PR, and I like the new
submit text better, so getting it back in.

edgeworthe commented on README.md:
```diff
@@ -61,12 +61,6 @@ To provide your own user list, replace the number on line 9 of the `gatsby-confi
-In order to facilitate the nominations of new engineers to follow, the nominate page allows users to send a DM directly to the twitter account of your choice.
-
-On line 16 of the `nominate.js` file, replace the variable `twitterIdForFollowingList` with the twitter id of your desired account.
-
```
we should probably add this back in at some point; I didn't notice previously that reverting the commit removed this feature
